core/src/foundations/ionic.root.scss

```diff
@@ -59,6 +59,7 @@
   --ionic-space-600: 24px;
   --ionic-space-700: 28px;
   --ionic-space-800: 32px;
+  --ionic-space-850: 34px;
```

Comment:
Is there a reason we are adding 850 but not 750, 650, etc? Are we just adding tokens as we need them?

Reply:
I spoke with Beth and she said we can review all of the tokens once they are implemented. 👍